test/commands/scanner/rule/remove.test.ts

```diff
@@ -48,8 +52,8 @@ describe('scanner:rule:remove', () => {
 		describe('Rule Removal', () => {
 			describe('Test Case: Removing a single PMD JAR', () => {
 				removeTest
-					// We'll wait three seconds then send in a 'y', to simulate the user confirming the request.
-					.stdin('y\n', 3000)
+					// We'll wait a few seconds then send in a 'y', to simulate the user confirming the request.
+					.stdin('y\n', 5000)
```

Comment:
With the addition of the new engine, `scanner:rule:remove` can occasionally take a bit more than 3 seconds to reach the "are you sure?" prompt. When that happens, the "y" is piped in too early, so the operation hangs until it times out and fails the tests.
Upping the time before piping "y" to 5 seconds should make these tests way less flaky.